src/System.Security.Cryptography.Pkcs/tests/SignedCms/CmsSignerTests.cs

```diff
@@ -1,24 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@bartonjs, this test file wasn't being included and the test doesn't pass.  Ok to delete?

Reply:
I fixed the compilation errors in it and opened https://github.com/dotnet/corefx/issues/30257 for the test failure.